### PR TITLE
[stable10] Changelog update for upcoming 10.0.8

### PR DESCRIPTION
**Pull request by PVince81**
38baf9389536fb01f77f96dfff41c36bcdb96b7f..3eb5053ecb6ee2e96e7639e2c6920d991a72598c

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,40 +5,88 @@ All notable changes to this project will be documented in this file.
...
 - Added new API events for commenting actions - [#30142](https://github.com/owncloud/core/issues/30142)
+- Added "register notifier" event for use with the notification emails feature - [#30613](https://github.com/owncloud/core/issues/30613)
+- Added group option to files:scan command - [#30615](https://github.com/owncloud/core/issues/30615)
+- Added warning if no files to process in occ files:transfer-ownership command - [#30612](https://github.com/owncloud/core/issues/30612)
+- Added user:modify command to core - [#30652](https://github.com/owncloud/core/issues/30652)
+- Added config switch to enable fallback to http scheme when creating fed shares - [#30646](https://github.com/owncloud/core/issues/30646)
+- Added repair step for orphaned sub-shares - [#30695](https://github.com/owncloud/core/issues/30695)
+- Added symfony events for configuration changes (config.php and appconfig) - [#30788](https://github.com/owncloud/core/issues/30788) [#30937](https://github.com/owncloud/core/issues/30937)
+- Added symfony event to let apps resolving private links - [#30911](https://github.com/owncloud/core/issues/30911)
```
Comment (phil-davis) [inline CHANGELOG.md]: s/resolving/resolve

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,40 +5,88 @@ All notable changes to this project will be documented in this file.
...
+- Bump behat/mink-extension from 2.3.0 to 2.3.1 - [#30706](https://github.com/owncloud/core/issues/30706)
+- Bump league/flysystem from 1.0.42 to 1.0.43 - [#30704](https://github.com/owncloud/core/issues/30704)
 - Update composer in stable10 with versions as at 2018-02-07 - [#30390](https://github.com/owncloud/core/issues/30390)
 - Renamed SMB logging config.php settings from "wnd" to "smb" - [#30244](https://github.com/owncloud/core/issues/30244)
 - Improved error messages in user:delete command - [#30164](https://github.com/owncloud/core/issues/30164)
 - Allow avatars to be visible publicly for public calendar - [#30228](https://github.com/owncloud/core/issues/30228)
+- Validate email address in mail settings section - [#30315](https://github.com/owncloud/core/issues/30315)
+- Only decrypt users who have already logged in with decrypt-all occ command - [#30640](https://github.com/owncloud/core/issues/30640)
+- Replace usage of "create_function" in PHP - [#30714](https://github.com/owncloud/core/issues/30714)
+- Provisioning API can now propertly set default or zero quota - [#30755](https://github.com/owncloud/core/issues/30755)
```
Comment (phil-davis) [inline CHANGELOG.md]: s/propertly/properly

**Comment by PVince81** [comment]
Fixed and added a few more entries in af45f5b46050ccefecb6835929a243f61ab167e6 @phil-davis

**Review by phil-davis — APPROVED** [review]
The text looks good. Of course I won't know about items that might be missing ;)

**Pull request merged**